ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@available(*, deprecated, renamed: "IndeterminateProgressBarView")
+public typealias MSIndeterminateProgressBarView = IndeterminateProgressBarView
+
+/**
+Indeterminate progress bar view
+ */
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView {
```

Comment:
>IndeterminateProgressBarView [](start = 11, length = 28)

this control should be similar to UIActivityIndictorView that we should have public "startAnimating" "StopAnimating" "hideswhenstopped" "animating"  functions and properties.